File: 0014-Longest-Common-Prefix/0014.java
class Solution {
    public String longestCommonPrefix(String[] strs) {
        String prefix = "";
        // iterate the first word in strs
        for (int i = 0; i < strs[0].length(); i++) {
            char c = strs[0].charAt(i);
            // check every word in strs
            for (String str : strs) {
                if (str.indexOf(prefix + c) != 0) {
                    return prefix;
                }
            }
            prefix += c;
        }
        return prefix;
    }
}

File: 0014-Longest-Common-Prefix/0014.py
from typing import List

class Solution:
    def longestCommonPrefix(self, strs: List[str]) -> str:
        answer = ''
        # iterate the first word in strs
        for char in strs[0]:
            # check every word in strs
            for i in range(1, len(strs)):
                # index will throw error if it's not exist
                if strs[i].find(answer + char) != 0:
                    return answer
            answer += char
        return answer